docs: Clarify manual graceful shutdown in dev server (#51078)
vercel/next.js: docs/02-app/01-building-your-application/07-deploying/index.mdx

> Sometimes you might want to run some cleanup code on process signals like `SIGTERM` or `SIGINT`.

Reviewer: When self-hosting, you might want to run code when the server shuts down on `SIGTERM` or `SIGINT` signals.

> Sometimes you might want to run some cleanup code on process signals like `SIGTERM` or `SIGINT`.

Reviewer: ```suggestion
When self-hosting, you might want to run code when the server shuts down on `SIGTERM` or `SIGINT` signals.
```

> You can set the env variable `NEXT_MANUAL_SIG_HANDLE` to `true` and then register a handler for that signal inside your `_document.js` file. Please note that you need to register the env variable directly in the system env variable, not in the `.env` file.

Reviewer: ```suggestion
You can set the env variable `NEXT_MANUAL_SIG_HANDLE` to `true` and then register a handler for that signal inside your `_document.js` file. You will need to register the env variable directly in the `package.json` script, not in the `.env` file.
```

I wonder what "system env var" means here - package.json script?

> You can do that by setting the env variable `NEXT_MANUAL_SIG_HANDLE` to `true` and then register a handler for that signal inside your `_document.js` file. Please note that you need to register env variable directly in the system env variable, not in the `.env` file.

Reviewer 2: ```suggestion
You can set the env variable `NEXT_MANUAL_SIG_HANDLE` to `true` and then register a handler for that signal inside your `_document.js` file. Please note that you need to register the env variable directly in the system env variable, not in the `.env` file.
```
